doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -451,6 +456,7 @@ Bluetooth libraries and services
 
     * Salt size in the Fast Pair not discoverable advertising from 1 byte to 2 bytes, to align with the Fast Pair specification update.
     * The :kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_OBERON` Kconfig option is now the default Fast Pair cryptographic backend.
+    * The Fast Pair documentation with debug Fast Pair provisioning data obtained for development purposes.
```

Comment:
shouldn't we change this changelog entry as we no longer list the FP Models in the user guide?

You can find them now in the application and sample projects